pennylane/devices/tests/test_measurements.py

```diff
@@ -152,7 +153,8 @@ def circuit():
 class TestHamiltonianSupport:
     """Separate test to ensure that the device can differentiate Hamiltonian observables."""
 
-    def test_hamiltonian_diff(self, device_kwargs, tol):
+    @pytest.mark.parametrize("ham_constructor", [qml.Hamiltonian, qml.ops.LinearCombination])
```

Comment:
```suggestion
    @pytest.mark.parametrize("ham_constructor", [qml.ops.Hamiltonian, qml.ops.LinearCombination])
```
With new op math enabled, this parametrization would essentially be testing the same thing twice.